src/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import BarreDeRecherche from './component/BarreDeRecherche';
+import Images from './component/images';
+
+
+class App extends React.Component {
+  render(){
+    return (
+      <React.Fragment>
```

Comment:
@malvena15 Please justify the use of `class components` rather than `functional components`.

Reply:
It's more out of habit that I used the class components rather than the function components. By cons in the case of the application I did not use the states and I had small component it would have been more efficient to use the function